DSharpPlus.Commands/Processors/SlashCommands/SlashCommandConfiguration.cs

```diff
@@ -21,4 +22,25 @@ public sealed class SlashCommandConfiguration
     /// How to name parameters when registering or receiving interaction data.
     /// </summary>
     public IInteractionNamingPolicy NamingPolicy { get; init; } = new SnakeCaseNamingPolicy();
+
+    /// <summary>
+    /// Instructs DSharpPlus to always overwrite the command records Discord has of our bot on startup.
+    /// </summary>
+    /// <remarks>
+    /// This skips the startup procedure of fetching commands and overwriting only if additions are detected. While
+    /// this may save time on startup, it also makes the library less resistant to unrecognized command types or
+    /// structures it cannot correctly handle. <br/>
+    /// Currently, removals are <i>not</i> considered a reason to overwrite by default so as to work around an issue
+    /// where certain commands will cause bulk overwrites to fail.
+    /// </remarks>
+    public bool UnconditionallyOverwriteCommands { get; init; } = false;

```

Comment:
Should this stay as a bool or should we turn it into an enum? What other cases should we reasonably support?

Reply:
ideally it should cease to exist entirely once we have received feedback for the feature and are confident in this approach, but in any case this is a question we should answer once we have received feedback on this